README.md

```diff
@@ -3,12 +3,29 @@
 A demo app using [FrankenPHP](https://frankenphp.dev) that uses
 Symfony and API Platform.
 
+## Prerequisite
```

Comment:
Can we drop this section? Actually, there are no prerequisites, this project works even without FrankenPHP.

Reply:
Yes, but the goal is to test FrankenPHP, no? And we can't without docker.

Reply:
Docker is easier but entirely optional: https://github.com/dunglas/frankenphp/blob/main/docs/compile.md

Reply:
Done.